includes/tracks/events/class-wc-orders-tracking.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * WooCommerce Import Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of WooCommerce Orders.
+ */
+class WC_Orders_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public static function init() {
+		add_action( 'woocommerce_order_edit_status', array( __CLASS__, 'track_orders_edit_status_change' ), 10, 2 );
+		add_action( 'admin_init', array( __CLASS__, 'track_orders_view' ), 10 );
+	}
+
+	/**
+	 * Send a Tracks event when the Orders page is viewed.
+	 */
+	public static function track_orders_view() {
+		if ( '/wp-admin/edit.php' === $_SERVER['SCRIPT_NAME'] && 'shop_order' === $_GET['post_type'] ) {
```

Comment:
Better follow what Jeff said in https://github.com/woocommerce/woocommerce/pull/22794/files#r259010088
This `wp-admin` may change in some installations, so better use WP hooks instead.